frontend/src/bundles/common/components/upload-video/components/dropzone/video-dropzone.tsx

```diff
@@ -0,0 +1,150 @@
+import { Input } from '@chakra-ui/react';
+import { DndContext, useDroppable } from '@dnd-kit/core';
+import { type ChangeEvent, type DragEvent } from 'react';
+
+import { Box } from '~/bundles/common/components/components.js';
+import { useCallback, useRef } from '~/bundles/common/hooks/hooks.js';
+import { notificationService } from '~/bundles/common/services/services.js';
+
+import { DropzoneInstructions } from './components/components.js';
+import {
+    DROPZONE_ID,
+    MAX_FILE_SIZE,
+    MIN_VIDEO_DURATION,
+} from './libs/constants/constants.js';
+import {
+    DOM_EVENT,
+    ERROR_ID,
+    ERROR_MESSAGE,
+    ERROR_TITLE,
+    HTML_TAG,
+} from './libs/enums/enums.js';
+import { isFileSizeValid, isVideoFile } from './libs/helpers/helpers.js';
+
+type Properties = {
+    videoSource: string | null;
+    handleRemoveVideo: () => void;
+    handleSetVideo: (fileUlr: string) => void;
+};
+
+const VideoDropzone: React.FC<Properties> = ({
+    videoSource,
+    handleRemoveVideo,
+    handleSetVideo,
+}) => {
+    const inputFile = useRef<HTMLInputElement | null>(null);
+    const { setNodeRef } = useDroppable({
+        id: DROPZONE_ID,
+    });
+
+    const handleFiles = useCallback(
```

Comment:
`handleFiles` name is not concrete enough. Maybe something like `handleLoadFile` or `handleAddFile` will be better?